qiskit/pulse/instructions/set_frequency.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Set channel frequency.
+"""
+from typing import Optional
+
+from qiskit.pulse.channels import PulseChannel
+from qiskit.pulse.commands.instruction import Instruction
+
+
+class SetFrequency(Instruction):
+    """
+    Set the channel frequency. This command operates on PulseChannels.
+    A PulseChannel creates pulses of the form Re[exp(i 2pi f jdt + phase) d_j].
+    F is the frequency of the channel. The command SetFrequency allows
```

Comment:
Here and in the next line, to keep consistent, this `F` should probably be \`\`f\`\`